### PR TITLE
Add AI game tournament web page

### PR DESCRIPTION
**Pull request by nvaytet**
This is the instructions on how to participate in the AI game tournament.

Note that most of the images will not work yet because the game repository is still private.
Ideally, I would like to wait until Monday morning before making it public, but maybe you would like to have the images here working before you can review/accept the PR?

### CODE REVIEW

File: data/pages-content/tournament.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+We have created a video game that is designed to be played by a small python program, rather than a human.
+Conference attendees can participate in a tournament where they (either alone or in a team) each submit a bot that will play the game, and at the end of the conference, we will have a tournament session where everyone can come and watch our strange creations play against each other (either brilliantly or it may all go wrong!).
+
+## Disclaimer
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/320px-Flag_of_Ukraine.svg.png" width="50" />
+
+The topic of the game is a sci-fi war on a foreign planet, set far in the future. We understand that the topic of war could upset some people with the unfortunate events that are currently going on in Ukraine. We do not want to promote war, and we fully support Ukraine in their hardship. Development of the game was started before the invasion of Ukraine, it is just a coincidence that this is the game that was ready for the conference.
+We just want to make a fun event for EuroPython participants, we do not want to offend anyone.
```
Comment (rodrigogiraoserrao) [inline data/pages-content/tournament.md:22]: How about a neutral “we don't condone war and violence” disclaimer?
Sadly, there is more than one armed conflict in the world right now, so calling out one specific conflict doesn't make much sense to me. It's like the other conflicts don't matter.

Comment (nvaytet) [reply data/pages-content/tournament.md:22]: > It's like the other conflicts don't matter.

Yes, I was thinking about that also. I will change to make it shorter and not name Ukraine specifically.

Comment (rodrigogiraoserrao) [reply data/pages-content/tournament.md]: I think the disclaimer now is better!
We'll see what others say when they review the PR...

Hopefully I'll see you at the tournament :D

File: data/links.json [hunk 2]
```diff
@@ -165,4 +169,4 @@
       "path": "https://www.europython-society.org/privacy/"
     }
   ]
-}
+}
```
Comment (nvaytet) [inline data/links.json:172]:
```suggestion
}

```

**Comment by nvaytet** [comment]
Hmm, the page shows up in the events menu
Screenshot at 2023-07-15 10-27-23

But I'm getting a 404 error when trying trying to go to the tournament page. Anyone has any ideas?

**Comment by nvaytet** [comment]
I'm happy with how this looks now. We'll just have to see on Monday morning how it looks when I make the repositories public.
I will only open the sign-up form on Wednesday morning.

**Comment by nvaytet** [comment]
Seems to have worked now, thanks for fixing the builds. It all looks good to me. You can merge/publish whenever you want, it doesn't matter at this point if it is one day early. Thanks for all the help!

**Comment by hypha** [comment]
If you really don't mind, we can merge it now so we have one less thing to keep track of. And thank you for organising this! Really looking forward!

**Comment by nvaytet** [comment]
Yes just go ahead, one thing less to think about is good 👍

**Pull request merged**